test/group0_tables/test_group0_tables.py

```diff
@@ -0,0 +1,91 @@
+#
```

Comment:
I'd begin with the basic syntax/limitations CQL tests before jumping onto the randomized tests train. Can I derive a view on a group0 table? Can I use a UDF inside a group0  table expression? Can I add a secondary key? We should cover these things with tests (and of course the main use case - conditional updates)

Reply:
> Can I derive a view on a group0 table? Can I use a UDF inside a group0 table expression?

Ugh, far outside the scope of the project.

Reply:
These are must-have questions if we want to have it as a generally available feature. I don't mean we should support it - I mean we should address it in  the spec, document and test.

Reply:
Agree it's outside the scope, but it should be relatively easy to implement (as a follow up, relax) since it can all be done in one transaction.

Reply:
Added a comment about the language limitation in the test.